Rt 1.54 bgp_override_as_path_split_horizon_test.go

cprabha opened the pull request:
Hi,

This is initial commit for RT-1.54

dplore commented:
/fptest virtual

dplore commented:
/fptest virtual

dplore commented:
@cprabha please move this test to the feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test folder.

(The experimental folder is now deprecated)

dplore commented:
@cprabha this test does not pass on cptx.  See [logs from the cptx test](https://storage.cloud.google.com/featureprofiles-ci-logs-juniper/log-a29dcdb4-3ee4-4293-8da3-3ffc4f620285.txt) mentioned in this PR request.  

Here is a snippet from those logs. Please investigate:

```
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:288: 
Step #1 - "fp-presubmit":         *** Starting protocols on otg...
Step #1 - "fp-presubmit":         
Step #1 - "fp-presubmit":         
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:468: Validate session state and capabilities received on DUT using telemetry.
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Verifying BGP state.
Step #1 - "fp-presubmit": I0918 01:07:53.410345   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Waiting for BGP neighbor to establish...
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: BGP adjacency for 192.0.2.2: ESTABLISHED
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Waiting for BGP neighbor to establish...
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: BGP adjacency for 192.0.2.6: ESTABLISHED
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:247: Verifying BGP capabilities
Step #1 - "fp-presubmit": I0918 01:07:55.466682   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:472: Verify that the DUT accepts the route.
Step #1 - "fp-presubmit": I0918 01:07:59.505229   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:473: Installed prefixes mismatch: got 0, want 1
Step #1 - "fp-presubmit": I0918 01:08:03.542736   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:474: Sent prefixes mismatch: got 0, want 1
Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:476: Verify that the ATE Port2 receives the route.
Step #1 - "fp-presubmit": --- FAIL: TestBGPOverrideASPathSplitHorizon (999.62s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_DUT_interfaces (17.08s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_DEFAULT_network_instance (2.71s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_BGP_Neighbors (7.41s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Verify_port_status_on_DUT (6.04s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Verify_BGP_telemetry (6.09s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Baseline_Test_No_allow-own-in (134.46s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_1 (138.50s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_1_Occurrence:_65500_dutLocalAS1_65499 (74.60s)
Step #1 - "fp-presubmit":     --- FAIL: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_3_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499 (130.29s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_4_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499_(Should_be_rejected) (134.34s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_3 (343.37s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_1_Occurrence:_65500,_dutLocalAS1,_65499 (74.35s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_3_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499#01 (70.31s)
Step #1 - "fp-presubmit":     --- FAIL: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_4_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499_(Should_be_accepted) (134.32s)
Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_4 (283.08s)
Step #1 - "fp-presubmit": FAIL
Step #1 - "fp-presubmit": I0918 01:09:07.623301   39650 core.go:212] core file check found cores:
Step #1 - "fp-presubmit": 
Step #1 - "fp-presubmit": Delta Core Files by DUT: 
Step #1 - "fp-presubmit": DUT: dut
Step #1 - "fp-presubmit": *** PROPERTY: validator.core.end -> 
Step #1 - "fp-presubmit": Delta Core Files by DUT: 
Step #1 - "fp-presubmit": DUT: dut
Step #1 - "fp-presubmit": 
Step #1 - "fp-presubmit": *** Releasing the testbed...
Step #1 - "fp-presubmit": 
Step #1 - "fp-presubmit": *** PROPERTY: time.begin -> 1726620738
Step #1 - "fp-presubmit": *** PROPERTY: time.end -> 1726621747
Step #1 - "fp-presubmit": FAIL	github.com/openconfig/featureprofiles/feature/experimental/bgp/otg_tests/bgp_override_as_path_split_horizon_test	1009.417s


```

cprabha commented:
> @cprabha please move this test to the feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test folder.
> 
> (The experimental folder is now deprecated)

@dplore moved files to main bgp folder.

cprabha commented:
> ```
> Step #1 - "fp
> ```

@dplore I will check this and update it.

cprabha commented:
> @cprabha this test does not pass on cptx. See [logs from the cptx test](https://storage.cloud.google.com/featureprofiles-ci-logs-juniper/log-a29dcdb4-3ee4-4293-8da3-3ffc4f620285.txt) mentioned in this PR request.
> 
> Here is a snippet from those logs. Please investigate:
> 
> ```
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:288: 
> Step #1 - "fp-presubmit":         *** Starting protocols on otg...
> Step #1 - "fp-presubmit":         
> Step #1 - "fp-presubmit":         
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:468: Validate session state and capabilities received on DUT using telemetry.
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Verifying BGP state.
> Step #1 - "fp-presubmit": I0918 01:07:53.410345   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Waiting for BGP neighbor to establish...
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: BGP adjacency for 192.0.2.2: ESTABLISHED
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: Waiting for BGP neighbor to establish...
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:469: BGP adjacency for 192.0.2.6: ESTABLISHED
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:247: Verifying BGP capabilities
> Step #1 - "fp-presubmit": I0918 01:07:55.466682   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:472: Verify that the DUT accepts the route.
> Step #1 - "fp-presubmit": I0918 01:07:59.505229   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:473: Installed prefixes mismatch: got 0, want 1
> Step #1 - "fp-presubmit": I0918 01:08:03.542736   39650 deviations.go:104] Did not match any platform_exception [platform:{vendor:NOKIA}  deviations:{explicit_interface_in_default_vrf:true  interface_enabled:true} platform:{vendor:ARISTA}  deviations:{route_policy_under_afi_unsupported:true  omit_l2_mtu:true  interface_enabled:true  default_network_instance:"default"  bgp_set_med_requires_equal_ospf_set_metric:true}], returning default values
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:474: Sent prefixes mismatch: got 0, want 1
> Step #1 - "fp-presubmit":     bgp_override_as_path_split_horizon_test.go:476: Verify that the ATE Port2 receives the route.
> Step #1 - "fp-presubmit": --- FAIL: TestBGPOverrideASPathSplitHorizon (999.62s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_DUT_interfaces (17.08s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_DEFAULT_network_instance (2.71s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Configure_BGP_Neighbors (7.41s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Verify_port_status_on_DUT (6.04s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Verify_BGP_telemetry (6.09s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Baseline_Test_No_allow-own-in (134.46s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_1 (138.50s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_1_Occurrence:_65500_dutLocalAS1_65499 (74.60s)
> Step #1 - "fp-presubmit":     --- FAIL: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_3_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499 (130.29s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_4_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499_(Should_be_rejected) (134.34s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_3 (343.37s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_1_Occurrence:_65500,_dutLocalAS1,_65499 (74.35s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_3_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499#01 (70.31s)
> Step #1 - "fp-presubmit":     --- FAIL: TestBGPOverrideASPathSplitHorizon/Re-advertise_the_prefix_from_the_ATE_with_4_Occurrences:_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_dutLocalAS1,_65499_(Should_be_accepted) (134.32s)
> Step #1 - "fp-presubmit":     --- PASS: TestBGPOverrideASPathSplitHorizon/_Test_allow-own-as_4 (283.08s)
> Step #1 - "fp-presubmit": FAIL
> Step #1 - "fp-presubmit": I0918 01:09:07.623301   39650 core.go:212] core file check found cores:
> Step #1 - "fp-presubmit": 
> Step #1 - "fp-presubmit": Delta Core Files by DUT: 
> Step #1 - "fp-presubmit": DUT: dut
> Step #1 - "fp-presubmit": *** PROPERTY: validator.core.end -> 
> Step #1 - "fp-presubmit": Delta Core Files by DUT: 
> Step #1 - "fp-presubmit": DUT: dut
> Step #1 - "fp-presubmit": 
> Step #1 - "fp-presubmit": *** Releasing the testbed...
> Step #1 - "fp-presubmit": 
> Step #1 - "fp-presubmit": *** PROPERTY: time.begin -> 1726620738
> Step #1 - "fp-presubmit": *** PROPERTY: time.end -> 1726621747
> Step #1 - "fp-presubmit": FAIL	github.com/openconfig/featureprofiles/feature/experimental/bgp/otg_tests/bgp_override_as_path_split_horizon_test	1009.417s
> ```

This is fixed, we need to add deviation . please could you re run and check.

dplore commented:
/fptest virtual

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// bgpCreateNbr creates a BGP object with neighbors pointing to atePort1 and atePort2
+func bgpCreateNbr(t *testing.T, dut *ondatra.DUTDevice) *oc.NetworkInstance_Protocol {
```
This should use cfgplugins.NewBGPSession. 

For an example, see:
https://github.com/openconfig/featureprofiles/blob/520108e9704e82444726ff35d0687f09163f8994/feature/bgp/policybase/otg_tests/aspath_test/aspath_test.go#L221

cprabha replied:
@dplore  will update all review comments . thanks

cprabha replied:
@dplore  I did check cfgplugins and its very generic one. We need to have specific AS numbers for this test. Please let me know if is ok to have local config and validation functions for this test.

dplore replied:
Ok.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// verifyBgpTelemetry checks that the dut has an established BGP session with reasonable settings.
+func verifyBgpTelemetry(t *testing.T, dut *ondatra.DUTDevice, nbrsList []*bgpNeighbor) {
```
Please use cfgplugins.VerifyDUTBGPEstablished https://github.com/openconfig/featureprofiles/blob/520108e9704e82444726ff35d0687f09163f8994/internal/cfgplugins/bgp.go#L316

cprabha replied:
updated this one.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// configureOTG configures the interfaces and BGP protocols on an ATE.
+func configureOTG(t *testing.T, otg *otg.OTG) (gosnappi.BgpV4Peer, gosnappi.DeviceIpv4, gosnappi.Config) {
```
can be obsoleted by NewBGPSession

cprabha replied:
I am using specific AS numbers for this test and these tests have predefined AS numbers , hence using local functions.

dplore replied:
Ok.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// bgpClearConfig removes all BGP configuration from the DUT.
+func bgpClearConfig(t *testing.T, dut *ondatra.DUTDevice) {
```
please move this to cfgplugins/bgp package

cprabha replied:
This generic package is updated to cfgplugins.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// Verify BGP capabilities like route refresh as32 and mpbgp.
+func verifyBGPCapabilities(t *testing.T, dut *ondatra.DUTDevice, nbrs []*bgpNeighbor) {
```
Please move to cfgplugins/bgp package

cprabha replied:
Updated this one

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+	}
+}
+
+func advBGPRouteFromOTG(t *testing.T, args *otgTestArgs, asSeg []uint32) {
```
this function should have a comment / description

dplore replied:
This can be moved to internal/otg_helpers/otg_config_helpers

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// configreRoutePolicy adds route-policy config.
+func configureRoutePolicy(t *testing.T, dut *ondatra.DUTDevice, name string, pr oc.E_RoutingPolicy_PolicyResultType) {
```
replace with https://github.com/openconfig/featureprofiles/blob/520108e9704e82444726ff35d0687f09163f8994/internal/cfgplugins/bgp.go#L256

cprabha replied:
Same with this func,  statement conditions is not present in cfgplugins configureRoutepolicy func also it is not exported.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+// verifyOTGPrefixTelemetry is to Validate prefix received on OTG por2.
+func verifyOTGPrefixTelemetry(t *testing.T, otg *otg.OTG, wantPrefix bool) {
```
Move to https://github.com/openconfig/featureprofiles/tree/main/internal/otg_helpers/otg_validation_helpers

cprabha replied:
this function also specific to this test and not a generic one. please let me know if it is ok keep it local.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/README.md:
```diff
@@ -6,7 +6,8 @@ BGP Override AS-path split-horizon
+ATE Port1 (AS 65502) --- eBGP --------- DUT Port1 (DUT Local AS 65501)
+ATE Port2 (AS 65503) --- eBGP --------- DUT Port2 (DUT Local AS 64513)
 
 ## Procedure
```
For each test below, add numbered headings formatted like:

```
### RT-1.54.1  Test no allow-own-in)
###  RT-1.54.2  Test allow-own-as 1:
```
and so on

cprabha replied:
This is updated for all subtests

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+}
+
+func testSplitHorizonNoAllowOwnIn(t *testing.T, args *otgTestArgs) {
+	t.Log("Baseline Test No allow-own-in")
```
Here and throughout, update the log message for the test to match the subtest heading in the readme. (ie: 1.54.1  Test no allow-own-in)

cprabha replied:
This is updated.

dplore commented on feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go:
```diff
@@ -0,0 +1,581 @@
+	t.Log("Test allow-own-as 1, Enable allow-own-as 1 on the DUT.")
+	dutConfPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(args.dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	if deviations.BgpAllowownasDiffDefaultValue(args.dut) {
+		gnmi.Replace(t, args.dut, dutConfPath.Bgp().PeerGroup(peerGrpName1).AsPathOptions().AllowOwnAs().Config(), 2)
```
I'm not inclined to allow this deviation?  Why doesn't "1" work as specified in the OC Model?  Is juniper going to be fixing this soon?

```
/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/as-path-options/config/allow-own-as

description:
Specify the number of occurrences of the local BGP speaker's AS that can occur within the AS_PATH before it is rejected.
nodetype: leaf (rw)
type: uint8
default: 0
```

cprabha replied:
We have different default values and it is updated in latest releases to match with OC default value.

dplore commented:
/fptest virtual

dplore commented:
@cprabha will you be addressing the comments soon?

cprabha commented:
@Darren ***@***.***> , I have started updating them, will push changes soon.

Thanks,
Prabha.



Juniper Business Use Only
From: Darren Loher ***@***.***>
Sent: Friday, November 8, 2024 2:03 PM
To: openconfig/featureprofiles ***@***.***>
Cc: Prabha Chethan ***@***.***>; Mention ***@***.***>
Subject: Re: [openconfig/featureprofiles] Rt 1.54 bgp_override_as_path_split_horizon_test.go (PR #3432)

[External Email. Be cautious of content]


@cprabha<https://urldefense.com/v3/__https:/github.com/cprabha__;!!NEt6yMaO-gk!FzGve7lFFsD12yWm8dvgG0dUtMm9y94s3UOyd2bZG3ZPZzytJIRWp6oCEOvyc0-h4mAODBYXe2NBVqQemQyGl09Z$> will you be addressing the comments soon?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/openconfig/featureprofiles/pull/3432*issuecomment-2465818715__;Iw!!NEt6yMaO-gk!FzGve7lFFsD12yWm8dvgG0dUtMm9y94s3UOyd2bZG3ZPZzytJIRWp6oCEOvyc0-h4mAODBYXe2NBVqQemXCfkbIE$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AHCEB5RSI7QX7MUOLYCQJZTZ7UYIRAVCNFSM6AAAAABOKLAAASVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINRVHAYTQNZRGU__;!!NEt6yMaO-gk!FzGve7lFFsD12yWm8dvgG0dUtMm9y94s3UOyd2bZG3ZPZzytJIRWp6oCEOvyc0-h4mAODBYXe2NBVqQemYm6rCdA$>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

dplore commented:
/fptest all
